content/chainguard/chainguard-images/working-with-images/security-advisories/how-chainguard-issues/index.md

```diff
@@ -0,0 +1,78 @@
+
+## Stages of a Security Advisory
+
+
+### Step 1: A CVE is Disclosed
+
+All security advisories begin with the disclosure of a security vulnerability. The [CVE Project](https://www.cve.org/) coordinates the processing of reported vulnerabilities through a network of CVE Numbering Authorities (CNAs). CNAs assign CVE IDs to new entries, and they are then added to a CVE Catalog. Each catalog entry contains information such as what packages or components are affected by the vulnerability, their versions, and remediation procedures, if applicable. 
+
+
+### Step 2: Scanners Detect the CVE
+
+The [National Vulnerability Database (NVD)](https://nvd.nist.gov/), the U.S. government vulnerability repository, will pick up these CVE records and review them further. During this secondary review process, the CVE entry is enriched with details that scanners later use to identify affected software. This process can take some time, so there will be issued CVEs that have not yet been analyzed by the NVD. These CVEs pending review will be marked as such by the NVD, as shown in the following image.
+
+![NVD notice stating a vulnerability has not yet been analyzed](nvd-analyze.png)
+
+In addition to the NVD, other vulnerability databases, such as the [GitHub Advisory Database](https://github.com/advisories) (GHSA) and the [Go Vulnerability Database](https://vuln.go.dev/) are also referenced by vulnerability scanners.
+
+
+### Step 3: Advisory is Issued
+
+Once a CVE has been reviewed by the NVD, it will be picked up by vulnerability scanners and reported in any affected container images. Chainguard uses [Grype](https://github.com/anchore/grype), an open-source vulnerability scanner from Anchore, as its primary tool for vulnerability detection.
+
+The newly detected CVE is then moved into the next phase where it waits for a team member to assess it. A security advisory will be issued with the status of “Under Investigation” to alert downstream users that Chainguard is aware of its presence. Security advisories are issued per package, as one CVE may impact different packages in different ways. This security advisory will be updated over time, as shown in the next step.
+
+
+### Step 4: Advisory is Updated
+
+With an advisory issued for the package, further investigation is often needed to determine the impact of the CVE. In some cases, it will be determined that the CVE is not truly present in the package, therefore making it a [false positive vulnerability](/chainguard/chainguard-images/recommended-practices/false-results/). The associated security advisory would have its status updated to “Not Affected”, and further updates to the advisory would not occur.
```

Comment:
```suggestion
With an advisory issued for the package, further investigation is often needed to determine the impact of the CVE. In some cases, it will be determined that the CVE is not truly present in the package, therefore making it a [false positive](/chainguard/chainguard-images/recommended-practices/false-results/). The associated security advisory would have its status updated to "Not Affected", and further updates to the advisory would not occur.
```